packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -29,9 +30,27 @@ export interface ServerRegistration {
   bodyParserConfig?: OptionsJson | boolean;
   onHealthCheck?: (req: express.Request) => Promise<any>;
   disableHealthCheck?: boolean;
-  gui?: boolean;
+  gui?: ((req: express.Request) => Partial<GuiOptions>) | Partial<GuiOptions>;
 }
 
+export interface GuiOptions {
+  enabled: boolean;
+  playgroundSettings: Partial<ISettings>;
```

Comment:
@stubailo was interested in a `defaultQuery` option. This should probably include a query, variables, operationName, and headers. The playground settings could take precedence with a console.warn

Reply:
I've added in the GraphQL Tabs options, which support this by default. If we add more options in the future, we may want to have a graphqlPlayground field which specifies all of the settings we allow and do a check that exactly one `gui` option is configured, but I really don't feel the need to over-abstract something that only has one allowable GUI at the moment.